blood/src/main/java/cz/cuni/mff/d3s/blood/report/dump/ManualDump.java

```diff
@@ -0,0 +1,23 @@
+package cz.cuni.mff.d3s.blood.report.dump;
+
+import java.util.function.Supplier;
+
+public final class ManualDump implements DumpRegistration {
+    final String name;
+    final Supplier<byte[]> dumpProducer;
```

Comment:
Why are you using `byte[]` when most (currently all) usages generate text? What about (at least) making `String` (or `CharSequence`) an option through another `DumpRegistration` implementation?